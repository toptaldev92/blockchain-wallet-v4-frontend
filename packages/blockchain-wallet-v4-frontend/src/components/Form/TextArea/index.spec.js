import { shallow } from 'enzyme'
import React from 'react'
import TextArea from './index.js'
import toJson from 'enzyme-to-json'
jest.mock('blockchain-info-components', () => ({
  Text: 'text',
  TextAreaInput: 'text-area-input'
}))

describe('TextArea', () => {
  it('renders correctly', () => {
    const props = { meta: {} }
    const component = shallow(<TextArea {...props} />)
    const tree = toJson(component)
    expect(tree).toMatchSnapshot()
  })
})
