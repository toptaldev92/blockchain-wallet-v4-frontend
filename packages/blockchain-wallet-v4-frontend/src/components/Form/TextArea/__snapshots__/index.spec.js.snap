// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextArea renders correctly 1`] = `
<styled.div>
  <text-area-input
    errorState="initial"
    rows={3}
  />
</styled.div>
`;
